frontend/src/components/ShootWorkers/GWorkerGroup.vue

```diff
@@ -10,28 +10,25 @@ SPDX-License-Identifier: Apache-2.0
     :toolbar-title="workerGroup.name"
     placement="bottom"
   >
-    <template #activator="{ props: popoverProps }">
-      <v-tooltip
-        location="top"
-        :disabled="!machineImage.isDeprecated"
-        text="Machine image version is deprecated"
+    <template #activator="{ props }">
+      <v-chip
+        v-bind="props"
+        size="small"
+        class="cursor-pointer"
+        variant="tonal"
+        :color="chipColor"
       >
-        <template #activator="{ props: tooltipProps }">
-          <v-chip
-            v-bind="mergeProps(popoverProps, tooltipProps)"
-            size="small"
-            class="cursor-pointer ma-1"
-            variant="tonal"
-            :color="chipColor"
-          >
-            <g-vendor-icon
-              :icon="machineImage.icon"
-              :size="20"
-            />
-            <span class="px-1">{{ workerGroup.name }}</span>
-          </v-chip>
-        </template>
-      </v-tooltip>
+        <g-vendor-icon
+          :icon="machineImage.icon"
+          :size="20"
+        />
+        <span class="pl-1">{{ workerGroup.name }}</span>
+        <v-tooltip
+          location="top"
+          :disabled="!machineImage.isDeprecated"
+          text="Machine image version is deprecated"
```

Comment:
This is because of the refactoring.... also added an overall deprecation for the collapsed chip. However, it is kind of cumbersome to always get the machineImage object from the cloud profile store. Maybe we can attach it to the workergroups in the shootitem composable? But not in this PR...